server/src-lib/Hasura/Server/API/PGDump.hs

```diff
@@ -87,7 +87,7 @@ execPGDump b ci = do
 
     notifyTriggerRegex =
       let regexStr :: String =
-            "^CREATE TRIGGER \"?notify_hasura_.+\"? AFTER [[:alnum:]]+ "
+            "^CREATE TRIGGER \"?(notify_hasura_|hasura_).+\"? AFTER [[:alnum:]]+ "
```

Comment:
We need a comment here relating this to `pgIdenTrigger` (which should also be documented) and mentioning how I guess `notify_hasura_` is there for backwards compatibility